src/test/java/com/github/freeclimbapi/DefaultApiTest.java

```diff
@@ -52,15 +53,14 @@ public void setup() {
      */
     @Test
     public void buyAPhoneNumberTest() throws ApiException {
-
+        
         BuyIncomingNumberRequest buyIncomingNumberRequest = buyIncomingNumberRequestBuyIncomingNumberRequestTestValue();
-
+        
         IncomingNumberResult response = this.apiInstance.buyAPhoneNumber(buyIncomingNumberRequest);
         // TODO: test validations
```

Comment:
noticed this file had a lot of these comments, was wondering what they mean?

Reply:
Hey @katebrune, this is an older generation of the DefaultApiTest. The comments will be changed in future improvements.